python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/flow-tracking.j2

```diff
@@ -109,7 +117,15 @@ flow tracking sampled
       !
 {%             endif %}
       exporter {{ exporter.name }}
-{%             if exporter.collector.host is arista.avd.defined %}
+{%             if exporter.collectors is arista.avd.defined %}
+{%                 for collector in exporter.collectors | arista.avd.natural_sort %}
```

Comment:
Add a TODO here to remove `arista.avd.defined` check since we are using `arista.avd.natural_sort` we would't have to check `arista.avd.defined`